hail/python/hail/docs/tutorials/01-genome-wide-association-study.ipynb

```diff
@@ -992,14 +960,13 @@
     "mt = hl.variant_qc(mt)\n",
     "common_mt = mt.filter_rows(mt.variant_qc.AF[1] > 0.01)\n",
     "\n",
-    "pca_eigenvalues, pca_scores, _ = hl.hwe_normalized_pca(common_mt.GT)\n",
+    "_, pcs, _ = hl.hwe_normalized_pca(common_mt.GT)\n",
```

Comment:
I'd keep `eigenvalues` even if not used explicitly in the `gwas` endpoint. They should always be inspected.